aws/cloudformation/rdp_bastion.yaml

```
@@ -0,0 +1,492 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  This template creates a bastion host based on Mate AMI on a new VPC(customer).
+  This bastion host will be used to connect to the ROSA cluster, run oc commands
+  and view the OpenShift console.
```

Missing space before the parenthesis:

```suggestion
  This template creates a bastion host based on Mate AMI on a new VPC
  (customer).
  This bastion host will be used to connect to the ROSA cluster, run oc commands
  and view the OpenShift console.
```